themes/agnoster.zsh-theme

```diff
@@ -203,7 +203,7 @@ prompt_virtualenv() {
 # - am I root
 # - are there background jobs?
 prompt_status() {
-  local symbols
+  local -a symbols
```

Comment:
This doesn't throw an error, but if you really want to change it then let's merge it with the line below:
```zsh
local -a symbols=()
```

Reply:
Changed it as suggested.